packages/pkgsrc.rb

```diff
@@ -125,8 +118,11 @@ def self.postinstall
     puts
     puts "To search packages, execute 'pkglocate <string>'.".lightblue
     puts
+  end
+
+  def self.remove
+    puts
     puts "To completely remove pkgsrc and all packages, execute the following:".lightblue
-    puts "crew remove pkgsrc".lightblue
     puts "rm -rf #{CREW_PREFIX}/pkg #{CREW_PREFIX}/pkgsrc".lightblue
```

Comment:
Prompt to remove?

Reply:
I'mma work on a function for prompt-remove stuff, I think we can leave this as is for now.